tools/export/cdt/__init__.py

```diff
@@ -29,11 +30,19 @@ def generate(self):
         if not exists(join(self.export_dir,'eclipse-extras')):
             makedirs(join(self.export_dir,'eclipse-extras'))
 
+        templates = ['%s.tmpl' % (self.target.lower())] + \
+            ['%s.tmpl' % (label.lower()) for label
+            in self.toolchain.target.extra_labels] + \
+            ['%s.tmpl' % 'pyocd_settings']
```

Comment:
As there are no target-specific templates, could we use the following instead:

```python
templates = ["pyocd_settings.tmpl", "pyocd_settings_debug.tmpl", "pyocd_settings_program.tmpl"]
```

Reply:
@theotherjimmy, I suggest to have 3 default templates:  templates = ["pyocd_settings_debug.tmpl", "pyocd_settings_program.tmpl", "pyocd_settings_erase.tmpl"] .   As I understood, the specific parameters should go through JSON file.